### ISSUE TITLE
Atmospheric heating doesn't work quite as it should

### ISSUE THREAD

**Issue by Brianetta** [pioneerspacesim/pioneer#1342]
A Hammerhead Heavy Freighter can't have shields fitted, but that's OK, because it's nearly impossible to get it significantly warmed up. One pixel on the thermometer, perhaps.

No ship should be able to aerobrake from orbital speed to landing speed, on Earth, without shielding. None.

Atmospheric heating is directly proportional to surface area and the outside pressure, but proportional to the _square_ of the speed. Orbital speeds are very, very high.

In addition to this, the current behaviour is that the thermometer rises, and when it hits the top the ship explodes. Presumably the thermometer threw shards of glass into the hyperdrive or something... anyway, it might be better if the hull sustains damage when the temperature is above some threshold.  The ship then explodes due to the hull being damaged rather than due to the temperature itself.

**Comment by impaktor** [comment]
Something tells me this is outdated and should/could be closed. I can't remember who's worked on friction and atmospheric heating last, but maybe @WKFO @fluffyfreak

**Comment by Web-eWorks** [comment]
Nope, atmo heating is definitely 100% broken and I was the one to do it. It's still pending a full investigation, as I hacked in a temporary workaround when we merged in atmospheric flight. I'll put this on my backlog but it might be a bit before I get a chance.
